lnbits/core/views/generic.py

```diff
@@ -247,11 +246,19 @@ async def wallet(
 
 @generic_router.get("/withdraw", response_class=JSONResponse)
 async def lnurl_full_withdraw(request: Request):
-    user = await get_user(request.query_params.get("usr"))
+    usr_param = request.query_params.get("usr")
+    if not usr_param:
+        return {"status": "ERROR", "reason": "usr parameter not provided."}
+
+    user = await get_user(usr_param)
     if not user:
         return {"status": "ERROR", "reason": "User does not exist."}
 
-    wallet = user.get_wallet(request.query_params.get("wal"))
+    wal_param = request.query_params.get("wal")
+    if not wal_param:
+        return {"status": "ERROR", "reason": "wal parameter not provided."}
+
+    wallet = user.get_wallet(wal_param)
```

Comment:
This should be handled by a `user` and `wallet` dependency as it is used multiple times

Reply:
getting the right error format with an `HTTPException` would require additional error formatters, I guess we'll leave it that way